collectors/test-results/jira-xray/src/main/java/com/capitalone/dashboard/core/PluginConstants.java

```diff
@@ -0,0 +1,11 @@
+package com.capitalone.dashboard.core;
+
+/**
+ * This class is for Jira XRay Plugin Constants
+ */
+public class PluginConstants {
+    public static final String XRAY_VERSION="1.12.2";
```

Comment:
@kumarvarun1252  pass versions from prop file instead of hardcoding it

Reply:
This variable is removed